.github/CODEOWNERS

```diff
@@ -1314,6 +1314,10 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 /x-pack/test/security_solution_cypress/cypress/e2e/overview @elastic/security-detection-engine
 x-pack/test/security_solution_api_integration/test_suites/detections_response/exceptions  @elastic/security-detection-engine
 x-pack/test/security_solution_api_integration/test_suites/detections_response/rule_creation @elastic/security-detection-engine
+/x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/bundled_prebuilt_rules_package @elastic/security-detection-rule-management
```

Comment:
@WafaaNasr Thanks for the heads up on updating CODEOWNERS.

Just pinging you here cause the routes that you added for your folders look non-existent, missing the `/default_license` dir. Not sure if it works anyways, but worth a check.

Reply:
Thanks, Juan!!  Yes correct I updated them in this [PR](https://github.com/elastic/kibana/pull/170350/files/a6167c5abc44ff102cab493e70796e6b5265e85b#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R1314)